.circleci/config.yml

```diff
@@ -18,7 +18,6 @@ jobs:
             apt-get install python3-venv --assume-yes
       - run: npm install
       - run: npm run build
-      - run: npm run build-storybook
```

Comment:
Removed this line because it was causing our CI to fail. A possible fix is installing, removing, and updating different Storybook dependencies. While I could spend more time trying to figure out this part, considering that we rarely (if ever) use this feature on dev, as Netlify preview is available to check any UI changes, I decided to remove this from running on CI. The headache that Storybook has caused us over the years while not being of much use is not worth it.

Reply:
We should still try to fix it when there's more engineering time. Just opened a [ticket](https://github.com/mozilla/glean-dictionary/issues/1630) to track it.